tests/test_distro.py

```diff
@@ -85,6 +89,33 @@ def test_cli_json(self):
         command = [sys.executable, '-m', 'distro', '-j']
         assert ast.literal_eval(self._run(command)) == distro.info()
 
+    def test_cli_with_root_dir(self):
+        root_dir = os.path.join(RESOURCES, 'cli', 'fedora30')
+        command = [sys.executable, '-m', 'distro', '--root-dir', root_dir]
+        desired_output = 'Name: Fedora 30 (Thirty)\nVersion: 30\nCodename: \n'
+        assert desired_output == self._run(command)
+
+    def test_cli_with_root_dir_as_json(self):
+        import json
+        root_dir = os.path.join(RESOURCES, 'cli', 'fedora30')
+        command = [sys.executable, '-m', 'distro', '-j', '--root-dir', root_dir]
+        desired_output = '''
+        {
+            "codename": "",
+            "id": "fedora",
+            "like": "",
+            "version": "30",
+            "version_parts": {
+                "build_number": "",
+                "major": "30",
+                "minor": ""
+            }
+        }
+        '''
+        desired_output = json.loads(desired_output)
```

Comment:
I noticed we're parsing hard-coded JSON here while then comparing high-level objects. How about removing the JSON indirection layer by going straight to Python nested dictionaries:
```
desired_output = {
    'codename': '',
    'id': 'fedora',
    'like': '',
    'version': '30',
    'version_parts': {
        'build_number': '',
        'major': '30',
        'minor': '',
    },
}
```
As a side-effect, we can now re-introduce trailing commas for better diffs if those lines are touched in the future. What do you think?

Reply:
Sure, I was a tad lazy for this. !